sdk/rust/oak_tests/src/lib.rs

```diff
@@ -789,7 +789,7 @@ pub fn start<S: ::std::hash::BuildHasher>(
         let node_name = name.clone();
         let thread_handle = spawn(move || {
             set_node_name(node_name);
-            entrypoint()
+            entrypoint(0)
```

Comment:
Is `0` meant to mean "invalid handle", or is it just an arbitrary handle number that is irrelevant for the rest of the logic?

Reply:
It's invalid handle (for now); added a constant and a comment.